### DISCUSSION TITLE
.gml file, what is "Other" node?

### DISCUSSION THREAD

**Discussion by Better-L** [ep142/FoodMicrobionet#3] [Q&A] [answered]
Hi,
after the selection step in Shiny, I end up with a dataset of 1650 taxa after aggregation.
When I open in Gephi 0.9.2 the exported .gml file, among my nodes I find the OTU "Other" under the "name" column.
Furthermore, the same node (with the same weighted degree calculated in Gephi statistics) is present in the .gml file of the same dataset before aggregation (2292 taxa).

This confuses me because no"Other" lable is present in the "prev_ab_table" of the same dataset.

Which sequence reads define the "Other" node?

Thank you very much!
Luca

**Comment by ep142** [comment, marked as answer]
Thank you for letting me know about this potentially confusing issue. 
The export tab lets you export files without any filtering step. On the other hand, the tabs on the right (prev ab table, bar plot, etc.) perform a filtering operation by removing “taxa” named other (this means ASVs for which not even the domain was assigned), Eukaryotic sequences, Chloroplasts and Mitochondria.
By the way, if you want to perform graphical and statistical analysis, you are better off exporting the agg file and using the FMBNanalyzer script of .Rmd file. Both are much more flexible in terms of output and filtering options
I have updated the app manual and I will push the modification during the next update cycle
--
Prof. Eugenio Parente
Scuola di Scienze Agrarie, Forestali, Alimentari ed Ambientali
Università degli Studi della Basilicata
Viale dell’Ateneo Lucano, 10
85100 Potenza, Italy

***@***.***
http://web.unibas.it/parente

**Reply by Better-L** [reply]
Thanks for the prompt and clear answer
